.github/workflows/build_images.yml

```diff
@@ -50,35 +49,34 @@ jobs:
 
       - name: Deploy Resource Group
         working-directory: ./resources
-        run: az deployment sub create 
-            --name "resource-group-deployment-${{ github.run_id }}" 
-            --location "${{ env.LOCATION }}" 
-            --template-file ./templates/resourceGroup/main.bicep
-            --parameters name="${{ env.RESOURCE_GROUP_NAME }}" location="${{ env.LOCATION }}"
+        run: az deployment sub create
+          --name="resource-group-deployment-${{ github.run_id }}"
+          --location="${{ env.LOCATION }}"
```

Comment:
Did we have to have `=` sign here?

Reply:
I think you added equal signs because of hyphens in client secret. Now that we ensure there is no hyphen in the password, can you remove them all and test again?

Reply:
done

Reply:
Can you do the same for the rest of the files. I can see `=` usages in other places.

Reply:
python scripts require = however have removed from other places that dont need =